src/pca.jl

```diff
@@ -47,11 +47,27 @@ transform(M::PCA{T}, x::AbstractVecOrMat{T}) where {T<:Real} = transpose(M.proj)
 reconstruct(M::PCA{T}, y::AbstractVecOrMat{T}) where {T<:Real} = decentralize(M.proj * y, M.mean)
 
 ## show & dump
-
 function show(io::IO, M::PCA)
     print(io, "PCA(indim = $(indim(M)), outdim = $(outdim(M)), principalratio = $(principalratio(M)))")
 end
 
+function show(io::IO, ::MIME"text/plain", M::PCA)
+    print(io, "PCA(indim = $(indim(M)), outdim = $(outdim(M)), principalratio = $(principalratio(M)))")
+    ldgs = projection(M) * diagm(0 => sqrt.(M.prinvars))
+    rot = diag(ldgs' * ldgs)
+    ldgs = ldgs[:,sortperm(rot, rev=true)]
+    ldgs_signs = sign.(sum(ldgs, dims=1))
+    ldgs_signs[ldgs_signs .== 0] .= 1
+    ldgs = ldgs * diagm(0 => ldgs_signs[:])
+    print(io, "\n\nPattern matrix\n")
+    display(ldgs)
+    print(io, "\n")
+    print(io, "Importance of components:\n")
+    print(io, CoefTable(vcat(principalvars(M)', (principalvars(M) ./ tvar(M))', (cumsum(principalvars(M) ./tvar(M)))'),
+                        string.("PC", 1:length(principalvars(M))),                      # components in order
+                        ["Loadings", "Proportion explained", "Cumulative proportion"])) # row names
```

Comment:
"Loadings" is "SS loadings" from psych, right? Shouldn't that be specified? Also "proportion" should say of what (the variance). And how about adding "proportion of variance" and  "cumulative variance" which are in base R and psych?

Reply:
They are not "Loadings", they are "Eigenvalues" or "SS loadings" as R calls them.

Loadings would be 
```
projection(M).*sqrt.(principalvars(M))'
```

Reply:
@wildart You're definitely right. I apologize for the oversight.